src/library/scala/util/Using.scala

```diff
@@ -145,14 +216,9 @@ object Using {
     * @return the result of the operation, if neither the operation nor
     *         releasing the resource throws
     */
-  def resource[R: Resource, A](resource: R)(body: R => A): A = {
+  def resource[R: Releasable, A](resource: R)(body: R => A): A = {
```

Comment:
good idea.

originally, `Releasable` was called `Resource`, which made it confusing to pass as a parameter since there is already a parameter named `resource`. That's not a problem now though